lib/buffer.js

```diff
@@ -1562,7 +1562,8 @@ if (process.binding('config').hasIntl) {
   // Buffer instance.
   transcode = function transcode(source, fromEncoding, toEncoding) {
     if (!isUint8Array(source))
-      throw new TypeError('"source" argument must be a Buffer or Uint8Array');
+      throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'source',
+                                 ['buffer', 'uint8Array'], source);
```

Comment:
please capitalize Buffer and Uint8Array.

Reply:
Most of the `ERR_INVALID_ARG_TYPE` errors in `lib` are using lower case.
For example:
https://github.com/nodejs/node/blob/master/lib/buffer.js#L198
https://github.com/nodejs/node/blob/master/lib/buffer.js#L225
https://github.com/nodejs/node/blob/master/lib/util.js#L965

Here is just for unitarity. It may need another PR to capitalize them.

Reply:
I think we should follow the convention of the docs here (lower case for primitives and capitals for others..)

Reply:
Those existing uses really ought to have been capitalized, we just missed them on the review, unfortunately.